### ISSUE TITLE
Couverture qui apparaît transparente

### ISSUE THREAD

**Issue by Rouergue** [EditionsDuRouergue/lerouergue#477]
Le visuel de l'album LA LETTRE DU LAC apparaît avec un fond transparent alors que j'ai re-téléchargé hier le bon visuel avec fond blanc (en jpeg et non pas en png). Quid ?

![Capture d’écran 2022-10-04 à 14 43 19](https://user-images.githubusercontent.com/10072454/193822429-5f7652a9-846d-4ae0-97ab-89443ec3417e.png)

**Comment by Fchallier** [comment]
Bonjour Mathilde, 
 Alors l'image déposée sur votre serveur est la suivante  : 
![9782812623929](https://user-images.githubusercontent.com/9974750/193824015-cce62295-2202-4da6-8615-6ccf10b357db.jpg)

Je ne sais pas si c'est la bonne, mais par contre celle-ci a la même couleur de fond que votre site... Il me semble que c'est pour cette raison que ça donne l’impression que c'est un PNG.  As-tu une autre version avec du vrai blanc comme couleur de fond  ?
Comme ça je dépose la nouvelle couverture et ensuite je lance le système de mise à jour + suppression des images en cache. 

Bien à toi,

**Comment by Rouergue** [comment]
Oui, tu as raison le fond déf de la couv est un peu grisâtre. Pour éviter que ça donne un rendu transparent sur notre site, j'ai bidouillé la couv en rajoutant du blanc. Cf. ci-joint. Merci pour la mise à jour du serveur !

![9782812623929](https://user-images.githubusercontent.com/10072454/193830199-75150072-6b64-4482-93e3-d590522b71ae.jpg)

**Comment by Fchallier** [comment]
Voilà c'est normalement à jour,  j'ai lancé à nouveau une mise à jour et supprimé le cache des images. 

Par contre, il faudra penser à faire passer le message à l'archivage.  Car dès que le BAT imprimeur sera passé dans le découpe Sirroco de Mistral l'image suivante sera remplace. Il faudra vérifier que la couverture archivé aura bel et bien un fond différent. 

Bien à toi,
